site/public/css/calendar.css

```diff
@@ -252,6 +255,11 @@ a.cal-gradeable-status-link:focus {
     border: none;
 }
 
+.calendar-color-blue {
+    background-color: #218ffd;
+    color: var(--text-black) !important;
```

Comment:
Is the `!important` necessary here?

Reply:
Is there a way to change the CSS to avoid using the `!important`?  Overrides like this can get messy really fast if someone every decides to try to change the CSS again.